R/expr-expr.R

```diff
@@ -2862,8 +2862,10 @@ expr__rolling <- function(
 #' * …
 #' * `(t_n - window_size, t_n]`
 #'
-#' @param by This column must be of dtype Datetime or Date. Accepts expression
-#' input. Strings are parsed as column names.
+#' @param by Should be DateTime, Date, UInt64, UInt32, Int64, or Int32 data 
+#' type after conversion by [`as_polars_expr()`][as_polars_expr]. Note that the
```

Comment:
```suggestion
#' type after conversion by [as_polars_expr]. Note that the
```

Reply:
Done, I just kept the parenthesis otherwise the link doesn't work anymore

Reply:
Oh sorry, my mistake. Thanks.